.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2024-05-23 - CSS-only Input States
```

markdownlint-cli2 (0.20.0)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)